src/main/scala/com.snowplowanalytics.snowplow.storage.kinesis/s3/SinkApp.scala

```diff
@@ -137,7 +125,10 @@ object SinkApp extends App {
     val kinesis = connector.getConfig("kinesis")
     val kinesisIn = kinesis.getConfig("in")
     val kinesisRegion = kinesis.getString("region")
-    val kEndpoint = s"https://kinesis.${kinesisSinkRegion}.amazonaws.com"
+    val kEndpoint = kinesisRegion match {
+      case "cn-north-1" => "kinesis.cn-north-1.amazonaws.com.cn"
+      case _ => s"https://kinesis.${kinesisSinkRegion}.amazonaws.com"
+    }
```

Comment:
Looks like the logic for resolving the AWS China endpoint is replicated twice.  Can this be turned into a function rather than duplicating the code?